kronos-java/src/test/java/com/lyft/kronos/internal/ntp/SntpServiceTest.kt

```diff
@@ -69,8 +69,10 @@ class SntpServiceTest {
         whenever(sntpClient.requestTime("1.us.pool.ntp.org", TIMEOUT_MS)).thenReturn(mockResponse)
         whenever(sntpClient.requestTime("2.us.pool.ntp.org", TIMEOUT_MS)).thenReturn(mockResponse)
         whenever(mockResponse.currentTimeMs).thenReturn(-1)
+
         assertThat(sntpService.sync()).isFalse()
 
-        verify(sntpSyncListener, times(1)).onError(any(), any<IllegalStateException>())
+        verify(sntpSyncListener, times(1)).onError(eq("1.us.pool.ntp.org"), any<NTPSyncException>())
+        verify(sntpSyncListener, times(1)).onError(eq("2.us.pool.ntp.org"), any<NTPSyncException>())
```

Comment:
This is a bit weird. Updating Kotlin results in this:

```
org.mockito.exceptions.verification.TooManyActualInvocations: 
syncListener.onError(
    <any java.lang.String>,
    <any com.lyft.kronos.internal.ntp.NTPSyncException>
);
Wanted 1 time:
-> at com.lyft.kronos.internal.ntp.SntpServiceTest.throw error if response has negative time value(SntpServiceTest.kt:74)
But was 2 times:
-> at com.lyft.kronos.internal.ntp.SntpServiceImpl.sync(SntpService.kt:160)
-> at com.lyft.kronos.internal.ntp.SntpServiceImpl.sync(SntpService.kt:160)
```

Looked through the code and seems like `sync` goes through all servers, calling listener for each server. So the adjustment I’ve made makes sense but not sure why the Kotlin version resulted in a behavior change. Might be bytecode changes and how Mockito manipulates it.